### ISSUE TITLE
Iou loss on multi gpu

### ISSUE THREAD

**Issue by konyul** [agent-sgs/PillarNet#17]
I was trying to adopt DIOU loss on this repository to original centerpoint repository, so I replaced ops/iou_nms and relavent files on original centerpoint repository with new ones on this repository. I also compiled c++ files by running setup.py.
 When I add IoU loss, The training is fine on single gpu. But when I run the centerpoint with DIOU loss on multi gpu. loss.backward doesn't work. Is there any problem with this??

**Comment by agent-sgs** [comment]
Can you provide more information? We didn't have this problem before.

**Comment by konyul** [comment]
I think this issue is caused by using apex. Did you use apex to train the model on multi gpus?

**Comment by Konstantin5389** [comment]
when mask.sum() == 0, return pred_boxes.sum() * 0 can solve this problem.
